README.md

```diff
@@ -244,11 +244,11 @@ If you configure your installation to have the React SDK static content served f
 If the `MediaCo` app was imported to your Infinity server, a `MediaCoOAuth` OAuth 2.0 Client Registration record will have been imported as well. That record's clientId is currently referenced within sdk-config.json.  However, you can create your own OAuth 2.0 Client Registration record using the following procedure:
    * Create a new "Security/OAuth 2.0 Client Registration" record for your app
    * You might name it the same name as your applicaion
-   * Specify "Public" for the type of client (as browser apps are not able to prevent any "Client secret" from being compromised)
+   * Specify "Public" for the type of client
    * Select "Authorization Code" for the Grant type
    * Add a RedirectURI value based on the url used to access the deployed React SDK (e.g., http://localhost:3502/auth.html)
    * Enable the "Enable proof code for pkce" option
-   * Set the "Access token lifetime" for how long you want the logged in session to last.  Pega does not presently support the ability to referesh the token (for Public clients), so the user will have to reauthenticate again after this interval.
+   * Set the "Access token lifetime" for how long you want the logged in session to last before the token is refreshed.
```

Comment:
before the token is silently refreshed.  We recommend somewhere between 15 minutes and at most 60 minutes...so a value between 900 and 3600.

Reply:
Also need to specify to set the refresh token lifetime to the interval before the user should encounter a full re-authentication challenge.  We recommend somewhere between 18 hours (64800) and 24 hours (86400)